geomesa-blobstore/geomesa-blobstore-api/src/main/scala/org/locationtech/geomesa/blob/api/BlobstoreServlet.scala

```diff
@@ -0,0 +1,51 @@
+package org.locationtech.geomesa.blob.api
+
+import java.io.{File, FileOutputStream}
+
+import com.google.common.io.Files
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloDataStoreFactory}
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore
+import org.locationtech.geomesa.web.core.GeoMesaScalatraServlet
+import org.scalatra.{NotFound, BadRequest, Ok}
+
+import scala.collection.JavaConversions._
+
+class BlobstoreServlet extends GeoMesaScalatraServlet {
```

Comment:
Agreed that we should have a POST for uploading a file.  @ellisbjohns will be entering a ticket for that.  Skipping the 'canProcess' may or may not make sense.  We'll have to see how things develop.